packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -22,6 +22,7 @@ import BaseBox from '~components/Box/BaseBox';
 
 const OVERLAY_OFFSET: number = size['8'];
 const OVERLAY_PADDING: number = size['12']; // doesn't have to be exact. Just rough padding for floating ui to decide to show overlay on top or bottom
+const OVERLAY_ZINDEX = 1001;
```

Comment:
Can we use our zindex token? chaitanya had created for modal? because it might get overridden by our consumers as in the past?

Reply:
https://github.com/razorpay/blade/blob/da72785595e4b76ccbc5940b66b5d950a5aaedd2/packages/blade/src/components/Modal/modalTokens.ts#L21

We haven't created zIndex tokens yet 🤔

Reply:
Actually we also allow overriding the zIndex on modal now. We didn't add it in Dropdown earlier. I'll add it here.

Reply:
done